pkg/analyze/distribution.go

```diff
@@ -77,6 +81,21 @@ func ParseNodesForProviders(nodes []corev1.Node) (providers, string) {
 				foundProviders.minikube = true
 				stringProvider = "minikube"
 			}
+			if k == "node.kubernetes.io/instance-type" && v == "k3s" {
+				foundProviders.k3s = true
+				stringProvider = "k3s"
+			}
+			if k == "beta.kubernetes.io/instance-type" && v == "k3s" {
+				foundProviders.k3s = true
+				stringProvider = "k3s"
```

Comment:
this is a duplicate of the above

Reply:
To clarify, you're saying you just want to see it consolidated on the same line? The key is different.

Reply:
🤦